### PR TITLE
Create connect to api

### PR DESCRIPTION
**Pull request by Lembani**
### This PR contains the _refactored_ version of the `bookstore` app to render books from `API using Redux & Axios`.

> Highlights

- Used Axios for making API calls.
- Fetch book list from `Bookstore API` on Books page load.
- Refactor the `add book` and `remove book` features to persist changes in the server.
- + minor improvements.

### CODE REVIEW

File: src/redux/configureStore.js [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 import categories from './categories/categories';
+import logger from './logger';
 
 const reducers = combineReducers({
   booksReducer,
   categories,
 });
 
-const store = createStore(reducers);
+const store = createStore(reducers, applyMiddleware(logger, thunk));
```
Comment (mateo951) [inline src/redux/configureStore.js:12]: `redux-thunk` is added to the project as a package and used as a middleware ✔

File: src/redux/books/books.js [hunk 2]
```diff
@@ -42,4 +45,11 @@ export const addBook = (book) => ({ type: ADD_BOOK, book });
 
 export const removeBook = (id) => ({ type: REMOVE_BOOK, id });
 
-export default booksReducer;
+export const getBooks = () => (dispatch) => {
+  axios.get(baseUrl).then(
```
Comment (mateo951) [inline src/redux/books/books.js:49]: All [Bookstore API](https://www.notion.so/Bookstore-API-51ea269061f849118c65c0a53e88a739) calls are done with `axios` 🤓 Well done!

**Review by mateo951 — APPROVED** [review]
Hi @Lembani, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🔥
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Lembani** [comment]
Hi @mateo951, Thanks for approving my PR. I appreciate. 🙏🏾

**Pull request merged**